indico/modules/events/editing/views.py

```diff
@@ -12,4 +12,5 @@
 
 class WPEditing(WPEventManagement):
     template_prefix = 'events/editing/'
-    bundles = ('module_events.editing.js', 'module_events.editing.css')
+    # remove mathjax later on from here (it is just here to make the timeline work)
```

Comment:
`# TODO: ...` (otherwise we'll forget it for sure - i usually search only for "TODO")